docs(vs): Add options article (#1550)
telerik/blazor-docs: getting-started/vs-integration/options.md

---

> title: Options

Reviewer: I suggest changing the title so that you apply the same action-oriented style used by the rest of the titles in this docs section.
```suggestion
title: Configuring the VS Extensions
```
![image](https://github.com/telerik/blazor-docs/assets/60105689/b48d3cf9-daa6-4088-97bf-15a991bdf7c9)

> page_title: Options

Reviewer: ```suggestion
page_title: Configuring the VS Extensions - Visual Studio Integration
```

description: Learn more about the available configurations that the Visual Studio Extension provides.
slug: getting-started-vs-integration-options
position: 8
---

> # Options

Reviewer: ```suggestion
# Configuring the Telerik VS Extensions
```

The **Options** dialog enables you to configure the Telerik Visual Studio Extensions to best suit your needs.

> The dialog can be accessed through **Visual Studio >> Extensions >> Telerik >> Telerik UI for Blazor >> Options:**

Reviewer: ```suggestion
The dialog can be accessed through **Visual Studio** > **Extensions** > **Telerik** > **Telerik UI for Blazor** > **Options...**
```

> It can be accessed through the **Visual Studio >> Extensions >> Telerik >> Telerik UI for Blazor >> Options:**

Reviewer 2: Avoid ambiguity; "the" can be used if it if followed by adjective + noun, which is not the case here.

```suggestion
The dialog can be accessed through **Visual Studio >> Extensions >> Telerik >> Telerik UI for Blazor >> Options:**
```

![Telerik UI for Blazor Visual Studio Extensions open Options dialog](images/vs-extension-open-options.png)

> The **Options** dialog contains two sets of configurations that affect the **Telerik UI for Blazor** Visual Studio Extension:

Reviewer: Don't use bold in the product names, features, etc.
```suggestion
The **Options** dialog contains two sets of configurations that affect the Telerik UI for Blazor Visual Studio Extension:
```

* [Notifications](#notifications-options)
* [General](#general-options)

> The settings under the **General** category affect all of the installed **Telerik Visual Studio Extensions.**

Reviewer: ```suggestion
The settings under the **General** category affect all installed Telerik Visual Studio Extensions—including those for other development platforms.
```

## Notifications Options

> This setting controls whether a message will be displayed to indicate when a newer version of **Telerik UI for Blazor**  is available:

Reviewer: ```suggestion
This setting controls whether a message will be displayed to indicate when a newer version of Telerik UI for Blazor is available:
```

![Telerik UI for Blazor Visual Studio Extensions Options - Notifications](images/vs-extension-options-notifications.png)

## General Options

The **General Options** window includes the following configurations:

![Telerik UI for Blazor Visual Studio Extensions Options - General](images/vs-extension-options-general.png)

### Download Location

> This setting allows you to select the folder for downloading the **Telerik UI for Blazor** package. The default path is:

Reviewer: ```suggestion
This setting allows you to select the folder for downloading the Telerik UI for Blazor package. The default path is:
```

````

> C:\Users\username\AppData\Roaming\Telerik\Updates

Reviewer 2: ```suggestion
C:\Users\[username]\AppData\Roaming\Telerik\Updates
```

````

### Project Setup

> * Copy referenced assemblies to solution and integrate with source control - When enabled, the referenced assemblies will be copied to the solution when using Telerik wizards. `True` by default.

Reviewer: ```suggestion
* **Copy referenced assemblies to solution and integrate with source control**—`True` by default. When enabled, the referenced assemblies will be copied to the solution when using Telerik wizards.
```

### Project Upgrade Notifications for Detected Local Distributions

> * Suggest project upgrades for Telerik product versions available on my computer - `True` by default.

Reviewer: ```suggestion
* **Suggest project upgrades for Telerik product versions available on my computer**—`True` by default.
```

> * Suggest upgrades when an equal Dev release is detected on projects using a Trial - `True` by default.

Reviewer: ```suggestion
* **Suggest upgrades when an equal Dev release is detected on projects using a Trial**—`True` by default.
```
